packages/babel-preset/react.js

```diff
@@ -1,10 +1,12 @@
 module.exports = function shopifyReactPreset(api, options = {}) {
   const env = api.env();
 
-  const pragma = options.pragma || 'React.createElement';
-  const pragmaFrag = options.pragmaFrag || 'React.Fragment';
-  const transformReactConstantElements =
-    options.transformReactConstantElements ?? true;
+  const {
+    pragma = 'React.createElement',
+    pragmaFrag = 'React.Fragment',
```

Comment:
the only way this "could" be a problem is these two are empty string and we are no longer catching it.

Reply:
Can we stick this destructuring straight in the parameters to dodge this intermediate value?

```
function shopifyReactPreset(
  api,
  {
    pragma = "React.createElement",
    pragmaFlag = "React.Fragment",
    transformReactConstantElements = true,
  } = {}
) {}

```